src/content/en/2020/markup.md

```diff
@@ -0,0 +1,626 @@
+
+<figure markdown>
+| Element | Probability |
+|---|---|
+| `dir` | 0.0082% |
+| `rp` | 0.0087% |
+| `basefont` | 0.0092% |
+<figcaption>{{ figure_link(caption="Low probabilities of finding a given element in pages of the sample.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
+</figure>
+
+### Custom elements
+
+The 2019 edition of the Web Almanac handled this part [a little differently](../2019/markup#custom-elements), discussing several non-standard elements. However, we found it valuable to have a closer look at custom elements. How did we determine these? Roughly by looking at [their definition](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-core-concepts), notably their use of a hyphen. Let’s focus on the top elements, in this case elements used on ≥1% of all URLs in the sample:
+
+<figure markdown>
+| Element | Occurrences | Percentage |
+|---|---|---|
+| `ym-measure` | 141,156 | 2.22% |
+| `wix-image` | 76,969 | 1.21% |
+| `rs-module-wrap` | 71,272 | 1.12% |
+| `rs-module` | 71,271 | 1.12% |
+| `rs-slide` | 70,970 | 1.12% |
+| `rs-slides` | 70,993 | 1.12% |
+| `rs-sbg-px` | 70,414 | 1.11% |
+| `rs-sbg-wrap` | 70,414 | 1.11% |
+| `rs-sbg` | 70,413 | 1.11% |
+| `rs-progress` | 70,651 | 1.11% |
+| `rs-mask-wrap` | 63,871 | 1.01% |
+| `rs-loop-wrap` | 63,870 | 1.01% |
+| `rs-layer-wrap` | 63,849 | 1.01% |
+| `wix-iframe` | 63,590 | 1% |
+<figcaption>{{ figure_link(caption="The 14 most popular custom elements.", sheets_gid="770933671", sql_file="pages_element_count_by_device_and_custom_dash_elements.sql") }}</figcaption>
+</figure>
+
+These elements come from three sources: [Yandex Metrics](https://metrica.yandex.com/about) (`ym-`), an analytics solution we’ve also seen last year; [Slider Revolution](https://www.sliderrevolution.com/) (`rs-`), a WordPress slider, for which there are more elements to be found near the top of the sample; and [Wix](https://www.wix.com/) (`wix-`), a website builder.
+
+Other groups that stand out include [AMP markup](https://amp.dev/) with `amp-` elements like `amp-img` (11,700 cases), `amp-analytics` (10,256) and `amp-auto-ads` (7,621), as well as [Angular](https://angular.io/) `app-` elements like `app-root` (16,314), `app-footer` (6,745), and `app-header` (5,274).
+
+### Obsolete elements
+
+There are more questions to ask about the use of HTML, and one may relate to obsolete elements—elements like `applet`, `bgsound`, `blink`, `center`, `font`, `frame`, `isindex`, `marquee`, or `spacer`.
+
+In our mobile data set of 6.3 million pages, around 0.9 million pages (14.01%) contained one or more of these elements. Here are the Top 9, those elements that were used more than 10,000 times:
+
+<figure markdown>
+| Element | Occurrences | Percentage of pages with this element |
```

Comment:
Looks very lob-sided:

![Generated Table Screenshot](https://user-images.githubusercontent.com/10931297/98299233-44445980-1faf-11eb-91be-bf7472e3e311.png)

Do you need such a long header name as you've used Percentage elsewhere:

```suggestion
| Element | Occurrences | Percentage |
```

Reply:
Off the top of my head, I think in this place “Percentage” is not clear enough. Can we leave this for now and review this again later? I’d think it stands out enough so to draw attention.